src/ucp/rma/rma.inl

```diff
@@ -32,6 +33,24 @@ ucp_rma_send_request_cb(ucp_request_t *req, ucp_send_callback_t cb)
     return req + 1;
 }
 
+static UCS_F_ALWAYS_INLINE ucs_status_ptr_t
+ucp_rma_send_request(ucp_request_t *req, const ucp_request_param_t *param)
+{
+    ucs_status_t status = ucp_request_send(req, 0);
+
+    if (req->flags & UCP_REQUEST_FLAG_COMPLETED) {
+        ucp_request_imm_cmpl_param(param, req, status, send);
+    }
+
+    ucs_trace_req("returning request %p, status %s", req,
+                  ucs_status_string(status));
+    if (param->op_attr_mask & UCP_OP_ATTR_FIELD_CALLBACK) {
+        ucp_request_set_callback(req, send.cb, param->cb.send, param->user_data);
```

Comment:
user_data flag is not checked
can we unite code with tag send?

Reply:
yep, unified